docs/user/_includes/label-limits.md

```diff
@@ -1,5 +1,6 @@
-* Labels must conform to the [label formatting regular expression](index.md#label-format).
-* Each label must have a maximum of 60 bytes, when encoded as UTF-8.
+* Labels must conform to the [label format](index.md#label-format).
+* Each label must have a maximum of 71 bytes, when encoded as UTF-8.
+* Each label must only contain printable ASCII characters.
```

Comment:
Well, given it's ASCII and for that "characters" is bascially intuitive what do you think about just phrasing that as such? Might be easier for people to read and understand.

```suggestion
* Each label must have a maximum of 71 characters.
* Each label must only contain printable ASCII characters.
```

(though I know we use the "encoded as UTF-8" part in other places, so maybe keeping it consistent is the better call here. I let you decide)